sdk_example.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from determined.experimental import Determined
+
+d = Determined(master='latest-master.determined.ai:8080')
+
+config = {
+    'this': 'is_my_test_config'
+}
+
+context_dir = os.getcwd()
+
+experiment = d.create_experiment({}, context_dir)
+experiment.wait_for_completion()
+
+if experiment.status == 'COMPLETED':
```

Comment:
isn't this actually `state`?

Reply:
nvm I just saw 

```
    @property
￼    def status(self) -> str:
```

Reply:
I actually think making this a `@property` is a bad idea, because it looks like the value is already present.  I think `get_status()` would be better, since that would communicate to the user that calling it is going to trigger some network communication (and you should be saving the output in a variable if you are going to call it multiple times).